toolchain/mfc/test/test.py

```diff
@@ -18,6 +18,8 @@
 
 
 nFAIL = 0
+nPASS = 0
+nSKIP = 0
```

Comment:
Even if it's unclear how `nSKIP` will be used (I don't see any test cases right now that we will skip, but I could imagine some), it seems like we should keep this 'feature'